Week-1/Assignment-1/readme.md

```diff
@@ -0,0 +1,98 @@
+<h1>Exercise 1.1 :</h1>
```

Comment:
Why you have used html in .dm file?

Reply:
To format the view

Reply:
.md file doesn't know about HTML. please follow this https://medium.com/@saumya.ranjan/how-to-write-a-readme-md-file-markdown-file-20cb7cbcd6f. 

add a readme viewer extension to your vscode.

Reply:
Ok Sir